SU-SOE/stanford_soe_helper
# SOE-2787: fix block styling

# READY FOR REVIEW

# Summary
Styling fix for the Magazine Department block. See https://engineering.stanford.edu/faculty-research/departments/aeronautics-astronautics 

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch  -  7.x-2.x-mag-dept-styling
- Remove "well" class on existing block
- May need clear cache
- Navigate to /faculty-research/departments/aeronautics-astronautics
- Verify you see fixed styling as specified in [this ticket](https://stanfordits.atlassian.net/browse/SOE-2787)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2787
## Related PRs

## More Information

## Folks to notify



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
